bootstrapping_decision_making.md

```diff
@@ -1,33 +1,34 @@
 # Bootstrapping Decision Making Bodies
 
-As the new Jupyter governance model is approved and implemented, Jupyter Subprojects will have to establish a formal decision making body. There are a number of challenges associated with this:
-
-- We are moving from an informal model where most Subprojects do not have a formal decision making body, to the formal one described in the Decision Making Guidelines.
-- Most Subprojects do not have a well defined decision making body today.
-- In the new model, there is flexibility about the size of the formal decision making bodies. Each Subproject will need to also determine how large its formal decision making body is.
-- There is a wide range of sizes of Subprojects &mdash; some have a few contributors and others have many dozens.
-- We are proposing to establish new Subprojects for certain areas of Project Jupyter, that don’t have an existing history of informal governance.
-- The new governance model and [decision making guide](decision_making.md) is designed to support large, highly participatory decision making bodies. As such, even Subprojects that have a clear decision making body today, may wish to increase the size of that body to include more contributors.
-- Our current governance model (informal consensus by an informal group lacking clear boundaries) does not have an established mechanism for decisions like this. We believe it will be very difficult for Subprojects to elect a formal governing body and pick its size using informal consensus.
-- We wish to avoid each Subproject having layers of meta consensus or voting to establish a process for electing their decision making body.
-- We need to take existing Steering Council members into account. There may be Steering Council members that would like to be on the formal decision making body for a Subproject, even though they have not been active on the Suproject for a period of time.
+As the new Jupyter governance model is approved and implemented, we are moving from an informal Subproject governance model to a formal model. As part of this process, each Jupyter Subproject will need to establish a formal decision-making body, including deciding its size and who its members are. There are a number of challenges associated with this:
+
+- Most Subprojects do not currently have a well-defined decision process or decision-making body.
+- There is a wide range of Subproject community sizes and activity&mdash;some have only a few contributors and infrequent commits, while others have dozens of developers and daily activity.
+- We are proposing to establish new Subprojects for certain areas of Project Jupyter (or combining related areas) that do not have an existing history of informal governance.
```

Comment:
This is a bit of a chicken and egg if this decision making PR is adopted before the other governance documents. I may be interpreting the messages on the multiple PRs. The way I understand it we're seeding Subprojects (esp. New Subprojects) before the Subprojects are specified and accepted in the other governance document.